qiskit/visualization/state_visualization.py

```diff
@@ -70,20 +70,23 @@ def plot_state_hinton(
         MissingOptionalLibraryError: Requires matplotlib.
         VisualizationError: if input is not a valid N-qubit state.
 
-    Example:
+    Examples:
         .. jupyter-execute::
 
             from qiskit import QuantumCircuit
             from qiskit.quantum_info import DensityMatrix
             from qiskit.visualization import plot_state_hinton
-            %matplotlib inline
+            import numpy as np
```

Comment:
third party imports usually go before local modules.

Reply:
Didn't know that. I'll change it

Reply:
@1ucian0 should there be a new empty line in between?

Reply:
Imported the same way with #8355, that already got merged. Should I open another PR to correct this?

Reply:
@Guillermo-Mijares-Vilarino I can fix that together here: https://github.com/Qiskit/qiskit-terra/pull/8449

Reply:
For this PR (8353), can you please change it? I will fix others since I am going to change something anyway.

Reply:
Of course!

Reply:
Fixed with 84a210d